src/data/i18n/default/src/components/AssetsImagePreviewFilter/displayMessages.json

```diff
@@ -1,7 +0,0 @@
-[
```

Comment:
These two files were added before my `.gitignore` change affected them; we no longer store them in github (they're calculated on-the-fly) as of https://github.com/edx/studio-frontend/pull/149